server/lib/src/filter.rs

```diff
@@ -660,10 +660,10 @@ impl FilterComp {
             FilterComp::Or(filters) => {
                 // If all filters are okay, return Ok(Filter::Or())
                 // If any is invalid, return the error.
-                // TODO: ftweedal says an empty or is a valid filter
+                // @ftweedal says an empty or is a valid filter
                 // in mathematical terms.
                 if filters.is_empty() {
-                    return Err(SchemaError::EmptyFilter);
+                    return Ok(FilterComp::Or(vec![]));
```

Comment:
I think these should stay as errors. Is there a reason to make them valid here?

Reply:
'cause an empty `Or` is an empty `Or` and thus valid and thus "false"

Reply:
But if someone requests an empty or, shouldn't we tell them "hey you're query looks a bit weird?" Is there a use case you are trying to resolve here?

Reply:
I dunno, I was mainly just cleaning up TODOs 😄 I can roll it back if you think it's an error.

> TODO: ftweedal says an empty or is a valid filter in mathematical terms.

Reply:
Also gosh damn that's some *ancient* code.

Reply:
Done!